File: typescript/src/api/resources/changesets/types/ListChangesetsChangesetsRequestStatus.ts

export const ListChangesetsChangesetsRequestStatus = {
        Draft: "draft",
        ReadyForReview: "ready_for_review",
        Approved: "approved",
        Staging: "staging",
        Production: "production",
        Rejected: "rejected",
        RolledBack: "rolled_back"
    } as const;
export type ListChangesetsChangesetsRequestStatus = typeof ListChangesetsChangesetsRequestStatus[keyof typeof ListChangesetsChangesetsRequestStatus];

File: ruby/lib/Sikaru/changesets/types/list_changesets_changesets_request_status.rb
# frozen_string_literal: true

module Sikaru
  module Changesets
    module Types
      module ListChangesetsChangesetsRequestStatus
        extend Sikaru::Internal::Types::Enum

        DRAFT = "draft"
        READY_FOR_REVIEW = "ready_for_review"
        APPROVED = "approved"
        STAGING = "staging"
        PRODUCTION = "production"
        REJECTED = "rejected"
        ROLLED_BACK = "rolled_back"
      end
    end
  end
end
